packages/test/test-service-load/src/runner.ts

```diff
@@ -444,23 +452,29 @@ function scheduleOffline(
 				ds.goOffline();
 
 				await new Promise<void>((resolve) => setTimeout(resolve, offlineTime));
-				if (!container.closed) {
-					ds.goOnline();
-					printStatus(runConfig, "going online!");
-					return schedule();
+				if (container.closed) {
+					return undefined;
 				}
+				if (runConfig.loaderConfig?.enableOfflineLoad === true) {
```

Comment:
should this be probablity based, so we don't always close when offline, only sometimes.